torchci/pages/api/job_annotation/[repoOwner]/[repoName]/[jobId]/[annotation].ts

```diff
@@ -1,14 +1,19 @@
 import { NextApiRequest, NextApiResponse } from "next";
 import { getDynamoClient } from "lib/dynamo";
-
+import { unstable_getServerSession } from "next-auth";
+import { authOptions } from "pages/api/auth/[...nextauth]";
 export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse
 ) {
   if (req.method !== "POST") {
     return res.status(504).end();
   }
-
+  // @ts-ignore
+  const session = await unstable_getServerSession(req, res, authOptions);
+  if (!session) {
```

Comment:
To let us start using the feature

```suggestion
  if (!session || !(session.username in ["ZainRizvi", "clee2000", "huydhn", "kit1980", "malfet", "janeyx"]) {
```